CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing guidelines
```

Comment:
prefer copy-pasting the content from https://github.com/kubernetes/kubernetes-template-project/blob/main/CONTRIBUTING.md as is.

Reply:
Modified